src/main/java/com/titarenko/service/EmployeeValidator.java

```diff
@@ -0,0 +1,22 @@
+package com.titarenko.service;
+
+import com.titarenko.model.Employee;
+import java.time.ZoneId;
+
+public class EmployeeValidator implements Validator{
+
+    @Override
+    public boolean isValidEmployee(Employee employee) {
+        int checkName = 0;
+        for (char c : employee.getName().toCharArray()) {
+            if (Character.isLetter(c))
+                checkName++;
+        }
+        long timeInMillis = employee.getDateOfHire().atStartOfDay(ZoneId.systemDefault()).toInstant().toEpochMilli();
```

Comment:
Since we store the only day of hire (without hours and minutes), I guess, this will be enough:
```employee.getDateOfHire().isBefore(LocalDate.now()))```